src/gui/editorwidgets/qgsrelationreferencewidget.h

```diff
@@ -269,6 +268,14 @@ class GUI_EXPORT QgsRelationReferenceWidget : public QWidget
     //! unset the currently related feature
     void deleteForeignKeys();
 
+    /**
+      * Trigger save of the embedded referenced attribute form.
+      * Returns true on success or if no embedded form is present.
```

Comment:
```suggestion
      * Returns TRUE on success or if no embedded form is present.
```
so it's correctly parsed